iOS/AppboyReactBridge/AppboyReactBridge/AppboyReactBridge.m

```diff
@@ -359,6 +414,85 @@ - (ABKCardCategory)getCardCategoryForString:(NSString *)category {
   return cardCategory;
 }
 
+RCT_REMAP_METHOD(getContentCards, getContentCardsWithResolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
+  RCTLogInfo(@"getContentCards called");
+  resolve([self getMappedContentCards]);
+}
+
+RCT_EXPORT_METHOD(logContentCardClicked:(NSString *)idString) {
+  ABKContentCard * cardToClick = [self getContentCardById:idString];
+  RCTLogInfo(@"logContentCardClicked with idString %@", idString);
+  if (cardToClick) {
```

Comment:
Is it worth considering the case when a card with a given id is missing? Like extra handling or loging a debug warning?

Reply:
I don't think we will have cards without ID, if we encounter any kind of issue we can definitely log something